spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Show', type: :feature do
+  before :each do
+    @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from US.',
+                        posts_counter: 0)
+    @first_post = Post.create(author_id: @user.id, title: 'First Post', text: 'This is my first post.',
+                              comments_counter: 0, likes_counter: 0)
+    @like = Like.create(post_id: @first_post.id, author_id: @user.id)
+    visit user_post_path(@user, @first_post)
+  end
+
+  it 'I can see the posts title' do
+    expect(page).to have_content 'First Post'
+  end
+
+  it 'I can see who wrote the post' do
+    expect(page).to have_content 'Lilly'
+  end
+
+  it 'I can see how many likes it has' do
+    expect(page).to have_content('Like(s): 1')
+  end
+
+  it 'I can see the post body' do
+    expect(page).to have_content 'This is my first post.'
+  end
```

Comment:
- [ ] @TeAM Great work adding tests for the post-show page, however, some tests are missing, I have highlighted below the tests that are missing, kindly consider adding them
![image](https://github.com/umair-rao/New-Blog-App/assets/61408860/5e40eb84-8328-4f96-9006-c7d5dfc5c194)

Reply:
Please check as the test are already there.

![showPost](https://github.com/umair-rao/New-Blog-App/assets/109624418/5f776290-ed6a-48e9-8d24-d98b645c2cb2)